valhalla/mjolnir/osmpbfparser.h

```diff
@@ -53,7 +55,7 @@ enum Interest {
 };
 
 // Represents the key/values of an object
-using Tags = std::unordered_map<std::string, std::string>;
+using Tags = phmap::flat_hash_map<std::string, std::string>;
```

Comment:
this is likely the main speed up

Reply:
Turns out it was only part of it - the `unordered_set` use in `IsNotThruEdge` was a big culprit, due to all the adding/removing of nodes to the `expandset`.